CHANGELOG.md

```diff
@@ -22,17 +22,21 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 - Scroll to top functionality on footer
 - Added `/modules/util/web-storage-proxy.js` utility module.
 - Added `/modules/util/js-loader.js` utility module.
+- Adds ESLint check for `@todo` jsdoc syntax.
```

Comment:
These won't get merged in before the next beta release, should we move the existing ones to a pre-release section and start a new un-released section?